src/app/member/[memberIdentifier]/dashboard/page.tsx

```diff
@@ -24,6 +24,7 @@ import dayjs from 'dayjs'
 import Link from 'next/link'
 import { DisplayStudyStatus } from './display-study-status'
 import { UserName } from '@/components/user-name'
+import { theme } from '@/theme'
```

Comment:
It looks like you're doing this cause `useMantineTheme` can't be used in a server component. we may just want to add one thing to the `theme` [per the docs here](https://mantine.dev/theming/theme-object/#access-theme-outside-of-components)

Reply:
Actually - im realizing we can just use `c='charcoal.1'` simple as that! think we were overcomplicating bringing in the full theme object. also the checks complain about `theme` being potentially undefined anyway 🤔